simulator/src/main/java/com/hazelcast/simulator/worker/loadsupport/StreamerFactory.java

```diff
@@ -60,13 +60,27 @@ private StreamerFactory() {
         return new SyncMapStreamer<K, V>(map);
     }
 
+    public static <K, V> Streamer<K, V> getInstance(IMap<K, V> map, int concurrencyLevel) {
```

Comment:
If i delegate this method to one without concurrencyLevel variable, i need define *DEFAULT_CONCURRENCY_LEVEL = 1000* at *StreamerFactory* again and pass this variable to old method. Imho *DEFAULT_CONCURRENCY_LEVEL* should be defined just in *AbstractAsyncStreamer*